godotproject/enemy-Ant.gd

```diff
@@ -52,6 +52,8 @@ func begin_attack():
 			if RC.name == "Player":
 				attackState = STATE.detected_player_e
 				$ActionDelay.start()
+	
+	$AttackSoundPlayer.play(0)
```

Comment:
I think this should maybe go where we create the fireball in the `launch_attack` function - it looks like this code is trying to detect the player and then set a timer. @jDenyes can you confirm?

Reply:
Ok so I tried moving this to the launch_attack function but because of the action delay timer the sound effect is also delayed. It doesnt really line up with whats happening graphically so Im going to leave it here